### PR TITLE
Fix: Carousel widgets are not working correctly inside Nested Tabs [ED-9790]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/scss/frontend/widgets/image-carousel.scss [hunk 1]
```diff
@@ -21,8 +21,7 @@
 }
 
 // Before the image-carousel layout is calculated by JS. //
-.elementor-image-carousel-wrapper:not(.swiper-container-initialized),
-.elementor-image-carousel-wrapper:not(.swiper-initialized) {
+.elementor-image-carousel-wrapper:not(.swiper-container-initialized):not(.swiper-initialized) {
```
Comment (hein-obox) [inline assets/dev/scss/frontend/widgets/image-carousel.scss:24]: Unrelated fix.

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 2]
```diff
@@ -439,7 +439,7 @@ export default class NestedTabs extends Base {
 
 		if ( isOnlyTabPressed && this.tabTitleHasActiveContentContainer( currentTabTitleIndex ) && !! $firstFocusableItem ) {
 			event.preventDefault();
-			$firstFocusableItem.focus();
+			$firstFocusableItem.trigger( 'focus' );
```
Comment (hein-obox) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:442]: Fixing of existing linting warning.

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 3]
```diff
@@ -66,6 +66,7 @@
 			flex-grow: 1;
 			padding: var( --n-tabs-content-padding );
 			border-radius: var( --n-tabs-content-border-radius );
+			min-width: 0;
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:69]: This is the fix.
The elements were disappearing inside a flex element without min-width: 0.

File: tests/playwright/sanity/modules/nested-tabs/tests/carousel.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+const { expect } = require( '@playwright/test' );
+const { selectDropdownContainer, clickTab } = require( '../helper' );
+
+async function testCarouselIsVisibleWhenUsingDirectionRightOrLeft( page, editor, widgetId ) {
+	// Act.
+	await page.pause();
```
Comment (davseve) [inline tests/playwright/sanity/modules/nested-tabs/tests/carousel.js:6]: Please remove the `await page.pause();`

File: assets/dev/scss/frontend/widgets/image-carousel.scss (hunk 1 above)
Comment (davseve) [reply assets/dev/scss/frontend/widgets/image-carousel.scss:24]: What this fix relates to?

**Review by davseve — CHANGES_REQUESTED** [review]

File: assets/dev/scss/frontend/widgets/image-carousel.scss (hunk 1 above)
Comment (hein-obox) [reply assets/dev/scss/frontend/widgets/image-carousel.scss:24]: The separate not selectors don't work, because the selectors all always try, which shouldn't be like that. I noticed while working on the Nested Carousel task.

File: tests/playwright/sanity/modules/nested-tabs/tests/carousel.js (hunk 4 above)
Comment (hein-obox) [reply tests/playwright/sanity/modules/nested-tabs/tests/carousel.js:6]: Updated.

**Review by davseve — APPROVED** [review]

**Review by louiswol94 — APPROVED** [review]
